cinn/hlir/framework/graph_compiler.h

```diff
@@ -91,8 +91,9 @@ class GraphCompiler final {
   };
 
   struct CompileOptions {
-    std::string attached_code       = "";
-    bool with_instantiate_variables = false;
+    std::string attached_code             = "";
+    bool with_instantiate_variables       = false;
+    bool insert_buffer_handle_instruction = false;
```

Comment:
insert_buffer_handle_instruction -> 
with_buffer_handle_instruction_inserted 
or to_insert_buffer_handle_instruction ?

Reply:
change to with_buffer_handle_instruction_inserted